### PR TITLE
update the communication file

### PR DESCRIPTION
**Pull request by dadishimwe**
---
name: solution review 
about: A template PR for code review with a checklist
---

### Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [ ] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [ ] The function has correct type annotations

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

### CODE REVIEW

File: collaboration/communication.md [hunk 1]
```diff
@@ -9,39 +9,64 @@
...
-- name: _5h_;
-- name: _4h_;
-- name: _3h_;
+_### Availability for Calling_
+...
+
+| Member| Monday | Tuesday | Wednesday | Thursday | Friday  | Saturday | Sunday |
+|-------------|------------|------------|------------|------------|------------|------------|------------|
+| Dadi    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm |
+| Member 2    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
```
Comment (AhmedKhalifa7) [inline collaboration/communication.md:45]: You can add me, available all week 05:00 - 07:00 PM

File: collaboration/communication.md [hunk 2]
```diff
@@ -9,39 +9,64 @@
...
+| Member 6    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 7    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 8    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+
+_### How Many Hours Everyone Has Per Day on Average_
+...
+
+- **Dadi**: 8 hours
+- **Rama**: # 4 hours
+- **Member 3**: # of hours
```
Comment (AhmedKhalifa7) [inline collaboration/communication.md:58]: Ahmed: 4 hours

File: collaboration/communication.md [hunk 3]
```diff
@@ -9,39 +9,64 @@
...
+- **Member 7**: # of hours
+- **Member 8**: # of hours
+
+---
+
+This README is structured to ensure a clear understanding of our communication
+strategy and availability. Adjust the time frames and hours as necessary to suit
+the team's schedule.
 
 ## Asking for Help
```
Comment (AhmedKhalifa7) [inline collaboration/communication.md:71]: What about this part?

**Review by AhmedKhalifa7 — CHANGES_REQUESTED** [review]
Inquire group members on  availablity for calling and how many hours everyone has per day on average.

File: collaboration/communication.md (hunk 3 above)
Comment (dadishimwe) [reply collaboration/communication.md:71]: Ok

File: collaboration/communication.md (hunk 1 above)
Comment (RaedAlyan) [reply collaboration/communication.md:45]: @AhmedKhalifa7 you have to add yourself to this file not Dadi

File: collaboration/communication.md (hunk 2 above)
Comment (RaedAlyan) [reply collaboration/communication.md:58]: the same thing, you have to add yourself

**Review by RaedAlyan — APPROVED** [review]
this PR looks good for me

File: collaboration/communication.md (hunk 1 above)
Comment (AhmedKhalifa7) [reply collaboration/communication.md:45]: Understood

**Review by AhmedKhalifa7 — APPROVED** [review]

**Review by AhmedKhalifa7 — DISMISSED** [review]
Ship it 🐿️

**Review by RaedAlyan — APPROVED** [review]
this PR looks fine to me!

**Comment by MohammadRAlSalloum** [comment]
Great Work!
Aprroved.

**Review by MohammadRAlSalloum — APPROVED** [review]
Nicely done!
